example/rails/config/environments/development.rb

```diff
@@ -2,6 +2,7 @@
   # Settings specified here will take precedence over those in config/application.rb.
   config.nylas_app_id = 'YOUR_APP_ID'
   config.nylas_app_secret = 'YOUR_APP_SECRET'
+  config.nylas_api_server = nil
```

Comment:
Hey @khamidou without specifying that variable I get this: http://i.imgur.com/5phVZUa.png . Does this not happen for you?